src/compressor.h

```diff
@@ -19,7 +19,13 @@ bool CompressScript(const CScript& script, std::vector<unsigned char> &out);
 unsigned int GetSpecialScriptSize(unsigned int nSize);
 bool DecompressScript(CScript& script, unsigned int nSize, const std::vector<unsigned char> &out);
 
+// Defined for 0 <= nAmount <= MAX_MONEY.
```

Comment:
Please use a doxygen compatible comment for documenting functions, see https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-doxygen-compatible-comments

Reply:
Good point. Fixed! Thanks!